specs/trust-store-trust-policy.md

```diff
@@ -329,14 +329,20 @@ Notary Project allows user to execute custom validations during verification usi
         1. Validate that the timestamp signing algorithm satisfies [algorithm requirements](./signature-specification.md#signature-algorithm-requirements).
         1. Validate the `signing-certificate`([RFC-2634](https://tools.ietf.org/html/rfc2634)) or `signing-certificate-v2`([RFC-5126](https://tools.ietf.org/html/rfc5126#section-5.7.3.2)) attribute of timestamp CMS.
         1. Validate that timestamp certificate and certificate chain leads to a trusted TSA certificate as per value configured in `trustStore`.
-        1. Validate timestamp certificate and certificate chain revocation status  using [certificate revocation evaluation](#certificate-revocation-evaluation) section.
+        1. Validate timestamp certificate and certificate chain revocation status using [certificate revocation evaluation](#certificate-revocation-evaluation) section.
         1. Retrieve the timestamp's time from `TSTInfo.genTime`.
         1. Retrieve the timestamp's accuracy.
         If the accuracy is explicitly specified in `TSTInfo.accuracy`, use that value.
         If the accuracy is not explicitly specified and `TSTInfo.policy` is the baseline time-stamp policy([RFC-3628](https://tools.ietf.org/html/rfc3628#section-5.2)), use accuracy of 1 second.
         Otherwise, use an accuracy of 0.
         1. Calculate the timestamp range using the lower and upper limits per [RFC-3161 section 2.4.2](https://tools.ietf.org/html/rfc3161#section-2.4.2) and store the limits as `timeStampLowerLimit` and `timeStampUpperLimit` variables respectively.
-    1. Validate that the time range from `timeStampLowerLimit` to `timeStampUpperLimit` timestamp is entirely within the signing certificate and certificate chain's validity period. If the validation passes, continue to the next step. Else fail this step.
+    1. Validate that the time range from `timeStampLowerLimit` to `timeStampUpperLimit` is entirely within the signing certificate and certificate chain's validity period. If the validation passes, continue to the next step. Else fail this step.
+
+    If under signing scheme [`notary.x509.signingAuthority`](./signing-scheme.md/#notaryx509signingauthority):
+    1. Check for the `Authentic Signing Time` signed attribute. If it does not exist, fail this step.
+    1. Store the `Authentic Signing Time` in variables `timeStampLowerLimit` and `timeStampUpperLimit`.
+    1. Validate that the time range from `timeStampLowerLimit` to `timeStampUpperLimit` is entirely within the signing certificate and certificate chain's validity period. If the validation passes, continue to the next step. Else fail this step.
```

Comment:
nit: this step can be combined for both signing schemes.

Reply:
Yeah, I tried to combine them in my first version. However, this part is explicitly mentioned when signing under signing scheme `notary.x509`. Thus, a combined step made the numbering of steps not trivial. I guess we could keep it as is?